test/plugins/features/movies/controller.test.js

```diff
@@ -4,10 +4,24 @@ const Knex = require('../../../../lib/libraries/knex');
 
 const Controller = require('../../../../lib/plugins/features/movies/controller');
 
+const LocationFactory      = require('../../../factories/location');
+const LocationMovieFactory = require('../../../factories/locations_movies');
+const MovieFactory         = require('../../../factories/movie');
+
+const testMovie1 = MovieFactory.build({ name: 'Zodiac', release_year: 2007 });
+const testMovie2 = MovieFactory.build({ name: 'Dawn of the Planet of the Apes', release_year: 2014 });
+
+const testLocation = LocationFactory.build();
+const testLocMov   = LocationMovieFactory.build({ movie_id: testMovie1.id, location_id: testLocation.id });
+
 describe('movie controller', () => {
 
   describe('create', () => {
 
+    beforeEach(async () => {
```

Comment:
similar comment: could you move this `beforeEach` block to the outer `describe` block? In this case, it would reduce the need to have multiple `beforeEach` blocks and copypasta. 

One thing to note though is that sometimes we don't want to have a `beforeEach` at the outermost describe block if we want to be more specific with how we want to run certain statements before given `describe` blocks

Reply:
copypasta... definitely using that one haha